include/seqan3/range/container/small_vector.hpp

```diff
@@ -333,13 +333,13 @@ class small_vector
         return &data_[sz];
     }
 
-    //!\copydoc small_vector::end().
+    //!\copydoc seqan3::small_vector::end()
     constexpr const_iterator end() const noexcept
     {
         return &data_[sz];
     }
 
-    //!\copydoc small_vector::end().
+    //!\copydoc seqan3::small_vector::end()
```

Comment:
```suggestion
    //!\copydoc seqan3::small_vector::cend()
```

Reply:
no because this is in the small_vector.hpp ! :D I just noticed myself. it has yo be end()